buildbot/configure.py

```diff
@@ -65,6 +68,15 @@ def do_configure(args):
     if args.enable_esimd_emulator:
         sycl_enabled_plugins.append("esimd_emulator")
 
+    if args.bldocl:
+        sycl_enable_opencl_cpu_rt_build = 'ON'
+
+    if args.bldfpgaemu:
+        sycl_enable_fpga_emu_build = 'ON'
+
+    if args.bldocl or args.bldfpgaemu:
+        llvm_external_projects += ';opencl-cpu'
+
```

Comment:
These components should be enabled when args.ci_defaults is true to build them in CI.

Reply:
@cdai2, please, address this comment.

I also recommend moving "Transform passes to support SYCL and OCL and LIT test cases for them" to a separate patch. It doesn't require changes in build scripts except CMake changes and will make it easier to review.

Please, provide instructions for reviewers how to review changes. GitHub is not able to display the full diff.

Reply:
I tried to enable opencl-cpu build under args.ci_defaults arguments according to my understanding.  Please correct me if it's not true.

Reply:
> As this PR is to upload these files for the first time. I have no idea to display the diff for it.

I think a major part of the diff is the content of `opencl/` directory. I suggest we keep changes in files outside of this directory in separate commits and you can ask code owners to review specific commits only and ignore the commits with changes in `opencl/` directory. I assume that @intel/ocl-cpu-rt-write team is already reviewed the content of the `opencl/` directory and can approve it. If not, the only idea I have is to pull the source code locally and review it locally.